client/signup/steps/site-info-collection/accordion-form-section.tsx

```diff
@@ -107,7 +107,7 @@ export default function AccordionFormSection( props: AccordionFormSectionProps )
 			</SectionHeader>
 			{ props.isExpanded && (
 				<SectionContent>
-					{ props.component }
+					{ props.component ? props.component : props.children }
```

Comment:
👍

Reply:
This gives a Typescript error: `Property 'children' does not exist on type 'AccordionFormSectionProps'.`

Reply:
I added this after the accordion refactor can you check again?